recipes/hyperspy_gui_traitsui/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hyperspy-gui-traitsui" %}
+{% set filename = "hyperspy_gui_traitsui" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "a2c8ca1917babee6c604a14988dc7907e8d13581c6307863e71d0cab05d27a74" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - qt 4.8.*
```

Comment:
So this pinning should occur in `traitsui` and not here. FWICT it is, but due to behavior from the `conda` solver it is not respected. Have opened PR ( https://github.com/conda-forge/pyface-feedstock/issues/10 ) to address this.